VoiceMeeterPlugin/Commands/Bases/BooleanBaseCommand.cs

```diff
@@ -124,12 +120,13 @@ private void GetNewSettings()
                             $"{(this.IsStrip ? "Strip" : "Bus")}[{hiIndex + this.Offset}].{this.Command}") == 1;
                 }
             }
-
-            this.ActionImageChanged();
+            if (loaded)
+                this.ActionImageChanged();
         }
-
+        protected bool loaded;
```

Comment:
It would be good, if you could move this variable to the others, also respect the naming convention please.